ppcp-gateway/angelleye-paypal-ppcp-common-functions.php

```diff
@@ -259,7 +263,7 @@ function angelleye_ppcp_get_mapped_billing_address($checkout_details, $is_name_o
 
 }
 
-if (!function_exists('angelleye_ppcp_get_mapped_shipping_address')) {
+if (!function_exists('  ')) {
```

Comment:
@Jaydeep7248 

I noticed in the pull request that the function check was altered from:

php

if (!function_exists('angelleye_ppcp_get_mapped_shipping_address')) {

to:

php

if (!function_exists(' ')) {

This change seems to have introduced a syntax issue, as the function name is missing in the function_exists check. This could lead to unexpected behavior since an empty function name will always return false, rendering the check ineffective. Please ensure the correct function name (angelleye_ppcp_get_mapped_shipping_address) is included in the function_exists check.

Let me know if you need any further clarification on this.